app/utils/validators.js

```diff
@@ -20,12 +20,14 @@ export const compulsoryProtocolValidUrlPattern = new RegExp('^(https?:\\/\\/)' /
 
 export const protocolLessValidUrlPattern = new RegExp(
   '^'
+  // localhost inclusion
+  + '(?:(?:localhost)(:\\d{2,5})?|'
   // user:pass authentication
   + '(?:\\S+(?::\\S*)?@)?'
   + '(?:'
   // IP address exclusion
   // private & local networks
-  + '(?!(?:10|127)(?:\\.\\d{1,3}){3})'
+  + '(?!(?:10)(?:\\.\\d{1,3}){3})'
```

Comment:
any link/resource to verify the validity of this check?

Reply:
The Loopback IP address ranges from 127.0.0.0 to 127.255.255.255 and as per the Issue the Loopback address(es) should be valid for testing purposes so I removed the 127 address from the exclusion list as it was just excluding any addresses beginning with 10/127.